src/components/Editor/GuestNameDialog.vue

```
@@ -87,7 +87,11 @@ const setGuestName = async () => {
 		const session = await update(guestName.value, connection.value)
 		loading.value = false
 		editing.value = false
-		localStorage.setItem('nick', guestName.value)
+		try {
+			localStorage?.setItem('nick', guestName.value)
```

I think optional chaining on `localStorage` doesn’t really help here – it’s always defined in the browser. The actual failures we need to handle are quota issues or disabled storage, which throw at `setItem`. Using `?.` would just skip the call if `localStorage` somehow wasn’t available, silently hiding problems, whereas `try/catch` actually surfaces real errors and lets us handle them properly.